core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -74,7 +74,7 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
   val pollTask: Cancellable =
     context.system.scheduler.schedule(pollInterval(), pollInterval(), self, Poll)(context.dispatcher)
 
-  var requests = Map.empty[TopicPartition, ActorRef]
+  var requests = Map.empty[ActorRef, RequestMessages]
```

Comment:
This map allows to have multiple `ActorRef` requesting for same topics-partition. What about checking for existing requests as part of handling of `RequestMessages`?